dbt_subprojects/hourly_spellbook/models/_sector/bridges/flows/evms/bridges_evms_deposits.sql

```diff
@@ -5,7 +5,6 @@
     , materialized = 'incremental'
     , file_format = 'delta'
     , incremental_strategy='merge'
-    , unique_key = ['deposit_chain','withdrawal_chain','bridge_name','bridge_version','bridge_transfer_id']
```

Comment:
this won't work. unique key is required for merge models. you either need to add columns or change the type of model

Reply:
ok i added an index for duplicates so i could readd unique keys with it. wdyt as is now?